src/audiomixerboard.cpp

```diff
@@ -619,14 +619,24 @@ CAudioMixerBoard::CAudioMixerBoard ( QWidget* parent, Qt::WindowFlags ) :
     bIsPanSupported      ( false ),
     bNoFaderVisible      ( true ),
     iMyChannelID         ( INVALID_INDEX ),
-    strServerName        ( "" )
+    strServerName        ( "" ),
+    eRecorderState       ( RS_UNDEFINED )
 {
     // add group box and hboxlayout
     QHBoxLayout* pGroupBoxLayout = new QHBoxLayout ( this );
     QWidget*     pMixerWidget    = new QWidget(); // will be added to the scroll area which is then the parent
     pScrollArea                  = new CMixerBoardScrollArea ( this );
     pMainLayout                  = new QHBoxLayout ( pMixerWidget );
 
+
+    setAccessibleName ( "Personal Mix at the Server groupbox" );
+    setWhatsThis ( "<b>" + tr( "Personal Mix at the Server" ) + "</b>: " +
+        tr ( "When connected to a server, the controls here allow you to set " ) +
+        tr ( "your local mix without affecting what others hear from you.") +
+        tr ( "<br/>The title shows - when known - the server name and whether it is recording, "
```

Comment:
It seems most users do not read help texts. They eventually just ask in the forum. Therefore it is no good idea to use these symbols. As written below, just show a clear text message and everybody knows whats going on. Then you do not need to explain anything here in the help text (or at least just a very short hint).

Reply:
Done.